public/app/plugins/panel/debug/DebugPanel.tsx

```diff
@@ -6,7 +6,7 @@ import { CursorView } from './CursorView';
 import { EventBusLoggerPanel } from './EventBusLogger';
 import { RenderInfoViewer } from './RenderInfoViewer';
 import { StateView } from './StateView';
-import { DebugPanelOptions, DebugMode } from './types';
+import { DebugPanelOptions, DebugMode } from './panelcfg.gen';
```

Comment:
If you [re-export the generated types](https://github.com/grafana/grafana/pull/62121/files#diff-02e7ec2462e8170844513cd3d1ffc8d1247586c62e68af891aef0ae0bfaa748fR5) from `types.ts` instead, you won't need to update them in other places.

Reply:
I think this is better -- simple enough to just update everywhere